code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -7,6 +7,11 @@
 	flags_inv = HIDEFACE
 	permeability_coefficient = 0.01
 	armor = list(melee = 65, bullet = 50, laser = 50, energy = 25, bomb = 50, bio = 100, rad = 50)
+	species_restricted = list("exclude", "Diona", "Wryn")

```

Comment:
Did this this way because after looking at the 'species_restricted' variable in game after simply porting the sprite, I saw these exact items populating the list with "Vox" among them. To get Vox to fit into the suit, I just removed the species from the restricted list and put it in here.